.github/workflows/merge-develop-to-main-workflow.yml

```diff
@@ -108,8 +108,8 @@ jobs:
 
       - name: Tag and Push Default Distroless RC Image
         run: |
-          docker tag splunk/splunk-operator:${{ github.event.inputs.release_version }}-RC${{ github.event.inputs.release_candiate_version }}-distroless splunk/splunk-operator-distroless:${{ github.event.inputs.release_version }}-RC-distroless
-          docker push splunk/splunk-operator:${{ github.event.inputs.release_version }}-RC-distroless
+          docker tag splunk/splunk-operator-rc:${{ github.event.inputs.release_version }}-distroless-RC${{ github.event.inputs.release_candiate_version }} splunk/splunk-operator-rc:${{ github.event.inputs.release_version }}-distroless-RC
```

Comment:
should it not be 
`splunk/splunk-operator:${{ github.event.inputs.release_version }}-distroless-RC`